utils/misc/misc.go

```diff
@@ -296,12 +298,35 @@ func RemoveFilePaths(filePaths ...string) {
 	}
 }
 
+// GetTempFolders returns all temporary folders.
+func GetTempFolders() (tempFolders []string) {
+	tmpDirPath, err := CreateTMPDIR()
+	if err != nil {
+		return
+	}
+
+	tempFolders = []string{
+		tmpDirPath,
+		fmt.Sprintf("%s/%s", tmpDirPath, "rudder-async-destination-logs"),
```

Comment:
maybe we maintain a map of these tmp paths and make sure we use them as variables in all places we use them. So that we would not miss adding any new path introduced in code

Reply:
Sure.

Reply:
Done.